hail/python/hail/experimental/datasets.py

```diff
@@ -0,0 +1,294 @@
+
+import json
+import hail as hl
+
+
+def load_dataset(dataset_name,
+                 reference_genome,
+                 config_file='gs://hail-datasets/hail_datasets.config.json'):
+    """Load a Hail-formatted genetic dataset.
+
+    Example
+    -------
+
+    >>> # Load 1000 Genomes chromosome X MatrixTable with GRCh38 coordinates
+    >>> mt_1kg = hl.experimental.load_dataset('1000_genomes_phase3_chrX',   # doctest: +SKIP
+    ...                                       reference_genome='GRCh38')
+
+    >>> # Count rows and columns
+    >>> mt_1kg.count()  # doctest: +SKIP
+
+    .. code-block:: text
+
+        (3474116, 2504)
+
+    >>> # Inspect contents of the MatrixTable
+    >>> mt_1kg.describe()   # doctest: +SKIP
```

Comment:
can we delete this? it clogs up the doc, a short `Notes` section with a link to the `Datasets` page would be way better IMO. I can also make this change as a followup if you prefer.